CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## [1.9.11] - 2019-11-01
+### Unreleased
+- Add available node build pack 10 & 12.
+- Update node build pack to default on v12.
```

Comment:
No other changelog entry ends with a period. Please remove the dots for consistency's sake.